src/type/timer.f90

```diff
@@ -45,99 +47,164 @@ module xtb_type_timer
 
 contains
 
+!--------------------------------------------
+!> To initialize timer
+!--------------------------------------------
```

Comment:
These are visually distracting, lets keep the comments free from those and only include the actual content
```suggestion
!> To initialize timer
```